tomcat9/modules/jws-unpack-tomcat9/module.yaml

```diff
@@ -15,15 +15,15 @@ artifacts:
       target: jws-5.4.0-application-server-RHEL8-ppc64le.zip
       description: "Artifact is available on Customer Portal:"
       md5: c559fe34c54c0e80b591f7c973aeb256    
-    - name: jws-5.4.1.CR1-application-server.zip
+    - name: jws-5.4.1.CR1.1-application-server.zip
       target: jws-5.4.1-application-server.zip
       description: "Artifact is available on Customer Portal:"
       md5: ba1e153ddb72c9d79c2b269e1bfca50e
```

Comment:
@mdrafiur Please check again and add the correct md5sum.

Reply:
@bekionel Sorry, missed to update the md5sum. Updated. Thanks

Reply:
It's ok now, merging. Please don't forget that you need to open a PR in https://github.com/jboss-container-images/jboss-webserver-5-openshift-image for the 5.4.1 update as well.